internal/describer/crd_watcher.go

```diff
@@ -80,9 +83,14 @@ func (cw *DefaultCRDWatcher) Watch(ctx context.Context, watchConfig *config.CRDW
 
 	err := cw.objectStore.Watch(ctx, crdKey, handler)
 	if err != nil {
-		if err, ok := err.(*objectstore.AccessError); ok {
-			logger := log.From(ctx)
-			logger.Warnf("%s", err)
+		aErr, ok := err.(*internalErr.AccessError)
+		if ok {
+			found := cw.errorStore.Add(aErr)
+			// Log if we have not seen this access error before.
+			if !found {
+				logger := log.From(ctx)
+				logger.WithErr(aErr).Errorf("access denied")
```

Comment:
Should CRDs be consistent with other objects and show access denied at least once?

Reply:
Not sure what you mean? Do you mean for each CRD? We attempt to watch all CRDs, so we are checking our permission ```
apiextensions.k8s.io/v1beta1
CustomResourceDefinition
```

We don't actually have a list of CRDs and likely can't get one anyway. So not sure there is a way to get more detailed than just stating that `watch` for CRDs has failed.

Also let me know if I understood the question.

Reply:
Actually this is fine, I got my wires crossed between running an older version of octant to compare console output